### PR TITLE
Add check to is.sorted for descending-ordered arrays. closes #97

### LINKED ISSUE #97: "is.sorted", isn't descending order considered sorted?
The implementation of ```is.sorted``` return true only for array sorted in ascending order. I think for descending order, it should also returns true.

### PR DESCRIPTION
**Pull request by rclanan**
There is probably a better implementation for this but this works until a better one can be put in place.

### CODE REVIEW

**Comment by ironmaniiith** [comment]
@rclanan , I don't guess such a big implementation is required, we can just check the direction (whether ascending or descending) using the first two entries in the array and proceed.

**Comment by ironmaniiith** [comment]
@rclanan , any update on this.?

**Comment by rclanan** [comment]
I missed the initial response on this. Let me look at this comment.

File: is.js [hunk 1]
```diff
@@ -768,12 +768,40 @@
...
-        for(var i = 0; i < arr.length; i++) {
-            if(arr[i] > arr[i + 1]) return false;
-        }
-        return true;
+
+        return checkArrayAsc(arr) || checkArrayDesc(arr);
     };
 
+    function checkArrayAsc(arr) {
+      for(var i = 0; i < arr.length; i++) {
```
Comment (ironmaniiith) [inline is.js]: we can loop only till `arr.length - 1` instead of `arr.length` here and avoid the `undefined` check for next below.

**Comment by jdalton** [comment]
Moved to #220.

**Pull request closed without merging**